46sshd/module-setup.sh

```diff
@@ -62,7 +62,9 @@ install() {
     grep '^sshd:' /etc/passwd >> "$initdir/etc/passwd"
     grep '^sshd:' /etc/group  >> "$initdir/etc/group"
 
-    mkdir -p "$initdir/var/empty/sshd"
+    # Create privilege seperation directory
+    /usr/bin/install -d -m 755 "$initdir/var/empty/sshd"
+    /usr/bin/install -d -m 755 "$initdir/var/lib/empty"
```

Comment:
I'm not a fan having an extra directory show up in the initramfs image. For example when debugging stuff one would have to remember which directory is actually supposed to be used.

What about making this conditional, e.g.:

    if [ -d /var/empty/ssh ]; then
        mkdir -p -m 755 "$initdir/var/empty/sshd"
    else
        mkdir -p -m 755 "$initdir/var/lib/empty"
    fi